doc/user_guide/user_guide.md

```diff
@@ -263,15 +265,57 @@ Please notice that we convert Avro complex types to the JSON Strings. Use Exasol
 `VARCHAR(n)` column type to store them. Depending on the size of complex type,
 set the number of characters in the VARCHAR type accordingly.
 
-## Import From Kafka Cluster
+## Importing Raw JSON
+
+When specifying ``RECORD_FORMAT=JSON`` the connector expects a valid UTF-8
+serialized JSON record per message. 
+When using ``ÀS_JSON_DOC=true``, the record is inserted as a whole and 
```

Comment:
Do you have unicode `A` on `AS_JSON_DOC` parameter? Could you please change it to regular `A`?